includes/documents/class-wcpdf-sequential-number-store.php

```diff
@@ -81,7 +81,7 @@ public function init() {
 		require_once( ABSPATH . 'wp-admin/includes/upgrade.php' );
 		$result = dbDelta( $sql );
 		wcpdf_log_error(
-			"dbDelta log for database table {$this->table_name} creation query: {$result}",
+			"dbDelta log for database table {$this->table_name} creation query results: {$result}",
```

Comment:
according to the docblock for `dbDelta()`, it returns an array:
```
@return array Strings containing the results of the various update queries.
```
So if you want to print this you will need to convert this into a human readable string first.

What sort of error does it return when the table can't be created (for example in the original #292 issue, I guess by setting `STRICT_TRANS_TABLES`?)?
IMO it would be better if we only log errors and nothing else.

But I wonder if this is even possible, reading this: https://wordpress.stackexchange.com/questions/141971/why-does-dbdelta-not-catch-mysqlerrors

Did you not test this logging?

Reply:
I've seen Query Monitor show errors from `dbDelta()`, so it should be possible to catch them?

Reply:
I tough you wanted to log everything from the `array`, in that case I will do it better.

Reply:
> I've seen Query Monitor show errors from `dbDelta()`, so it should be possible to catch them?

I will check that.

Reply:
> I thought you wanted to log everything from the array, in that case I will do it better.

I'm not saying I don't, but in all cases you need to convert it to a string first. Did you test this?

Reply:
No, but I see that I did a mistake, I will fix that.